examples/grpc.graphql

```diff
@@ -7,17 +7,10 @@ schema
 }
 
 type Query {
-  news: NewsData! @grpc(service: "news.NewsService", method: "GetAllNews", protoId: "news")
-  newsById(news: NewsInput!): News!
-    @grpc(service: "news.NewsService", method: "GetNews", body: "{{args.news}}", protoId: "news")
+  news: NewsData! @grpc(method: "news.news.NewsService.GetAllNews")
+  newsById(news: NewsInput!): News! @grpc(method: "news.news.NewsService.GetNews", body: "{{args.news}}")
```

Comment:
keep the example simple do this change for tests only,

Reply:
the example is broken and should be updated